tuf/repository_tool.py

```diff
@@ -2122,17 +2123,17 @@ def get_delegated_rolenames(self):
 
 
 
-  def delegate(self, rolename, public_keys, list_of_targets, threshold=1,
-      terminating=False, restricted_paths=None, path_hash_prefixes=None):
+  def delegate(self, rolename, public_keys, paths, threshold=1,
+      terminating=False, list_of_targets=None, path_hash_prefixes=None):
```

Comment:
To me it's not clear why `restricted_paths` has become `paths` and is now mandatory, whereas `list_of_targets` is now optional.